### PR TITLE
feat: Simplified ipv4 address

### PR DESCRIPTION
**Pull request by gsstoykov**
**Description**:
Ipv4 implementation had a useless abstraction. 

**Related issue(s)**:

Fixes https://github.com/hashgraph/hedera-sdk-go/issues/1036

**Notes for reviewer**:

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by 0xivanov — COMMENTED** [review]
Looks good, needs some more manual testing if it does not break anything during actual address book update.

File: ipv4_address.go [hunk 1]
```diff
@@ -21,47 +21,24 @@ package hedera
  */
 
 type IPv4Address struct {
```
Comment (0xivanov) [inline ipv4_address.go:23]: I think this stuct can be removed as well, it just holds the `address []byte` and does not really do anything.
For example the `endpoint` should call `endpoint.address.address` which is weird.

Comment (gsstoykov) [reply ipv4_address.go:23]: True, lets remove.

File: query.go [hunk 2]
```diff
@@ -203,7 +203,7 @@ func (q *Query) SetPaymentTransactionID(transactionID TransactionID) *Query {
 
 func (q *Query) execute(client *Client, e QueryInterface) (*services.Response, error) {
 	q.client = client
-	if client == nil || client.operator == nil {
```
Comment (0xivanov) [inline query.go:206]: note: removed since some queries don't require operator.

File: account_balance_query_e2e_test.go [hunk 3]
```diff
@@ -212,3 +214,31 @@ func TestIntegrationAccountBalanceQueryWorksWithHollowAccountAlias(t *testing.T)
...
+	client.SetTransportSecurity(true)
+
+	succeededOnce := false
+	for address, accountID := range client.GetNetwork() {
+
+		if !strings.HasSuffix(address, ":50212") {
+			t.Errorf("Expected entry key to end with ':50212', but got %s", address)
+		}
+
+		fmt.Println(address, accountID)
```
Comment (0xivanov) [inline account_balance_query_e2e_test.go:230]: can be removed

**Review by 0xivanov — APPROVED** [review]
LGTM

**Pull request merged**